# DP: Default to -fstack-protector-strong starting with Ubuntu 14.10.

Index: b/src/gcc/gcc.c
===================================================================
--- a/src/gcc/gcc.c
+++ b/src/gcc/gcc.c
@@ -872,7 +872,7 @@ proper position among the other output f
 
 #ifndef SSP_DEFAULT_SPEC
 #if defined(TARGET_LIBC_PROVIDES_SSP) && !defined(ACCEL_COMPILER)
-#define SSP_DEFAULT_SPEC "%{!fno-stack-protector:%{!fstack-protector-all:%{!ffreestanding:%{!nostdlib:-fstack-protector}}}} " FORMAT_SECURITY_SPEC
+#define SSP_DEFAULT_SPEC "%{!fno-stack-protector:%{!fstack-protector-all:%{!ffreestanding:%{!nostdlib:%{!fstack-protector:-fstack-protector-strong}}}}} " FORMAT_SECURITY_SPEC
 #else
 #define SSP_DEFAULT_SPEC FORMAT_SECURITY_SPEC
 #endif
